container-core/src/main/java/com/yahoo/metrics/Unit.java

```diff
@@ -5,53 +5,54 @@
  */
 public enum Unit {
 
-    AREA(BaseUnit.AREA),
-    BINARY(BaseUnit.BINARY),
-    BUCKET(BaseUnit.BUCKET),
-    BYTE(BaseUnit.BYTE),
-    BYTE_PER_SECOND(BaseUnit.BYTE, BaseUnit.SECOND),
-    CONNECTION(BaseUnit.CONNECTION),
-    DOCUMENT(BaseUnit.DOCUMENT),
-    DOCUMENTID(BaseUnit.DOCUMENTID),
-    FAILURE(BaseUnit.FAILURE),
-    FILE(BaseUnit.FILE),
-    FRACTION(BaseUnit.FRACTION),
-    HIT(BaseUnit.HIT),
-    HIT_PER_QUERY(BaseUnit.HIT, BaseUnit.QUERY),
-    INSTANCE(BaseUnit.INSTANCE),
-    ITEM(BaseUnit.ITEM),
-    MILLISECOND(BaseUnit.MILLISECOND),
-    NANOSECOND(BaseUnit.NANOSECOND),
-    NODE(BaseUnit.NODE),
-    PACKET(BaseUnit.PACKET),
-    OPERATION(BaseUnit.OPERATION),
-    OPERATION_PER_SECOND(BaseUnit.OPERATION, BaseUnit.SECOND),
-    PERCENTAGE(BaseUnit.PERCENTAGE),
-    QUERY(BaseUnit.QUERY),
-    QUERY_PER_SECOND(BaseUnit.QUERY, BaseUnit.SECOND),
-    RECORD(BaseUnit.RECORD),
-    REQUEST(BaseUnit.REQUEST),
-    RESPONSE(BaseUnit.RESPONSE),
-    RESTART(BaseUnit.RESTART),
-    SCORE(BaseUnit.SCORE),
-    SECOND(BaseUnit.SECOND),
-    SESSION(BaseUnit.SESSION),
-    TASK(BaseUnit.TASK),
-    THREAD(BaseUnit.THREAD),
-    VERSION(BaseUnit.VERSION),
-    WAKEUP(BaseUnit.WAKEUP);
+    BINARY(BaseUnit.BINARY, "Zero or one. Zero typically indicate \"false\" while one indicate \"true\""),
+    BUCKET(BaseUnit.BUCKET, "A chunk of documents managed by a distributor service"),
+    BYTE(BaseUnit.BYTE, "A collection of 8 bits"),
+    BYTE_PER_SECOND(BaseUnit.BYTE, BaseUnit.SECOND, "A unit of storage capable of holding 8 bits"),
+    CONNECTION(BaseUnit.CONNECTION, "A link used for communication between a client and a server"),
+    DOCUMENT(BaseUnit.DOCUMENT, "Vespa document, a collection of fields defined in a schema file"),
+    DOCUMENTID(BaseUnit.DOCUMENTID, "A unique document identifier"),
+    FAILURE(BaseUnit.FAILURE, "Failures, typically for requests or operations, nodes"),
```

Comment:
Is the ", nodes" at the end intentional? If so, I feel something is missing to explain.

Reply:
Thanks, updated.